pkg/kubelet/cm/dra/plugin/client.go

```diff
@@ -18,57 +18,27 @@ package plugin
 
 import (
 	"context"
-	"errors"
 	"fmt"
-	"io"
-	"net"
 	"time"
 
 	"google.golang.org/grpc"
 	grpccodes "google.golang.org/grpc/codes"
-	"google.golang.org/grpc/credentials/insecure"
 	grpcstatus "google.golang.org/grpc/status"
 	"k8s.io/klog/v2"
 
 	drapbv1alpha2 "k8s.io/kubelet/pkg/apis/dra/v1alpha2"
 	drapb "k8s.io/kubelet/pkg/apis/dra/v1alpha3"
 )
 
-const PluginClientTimeout = 10 * time.Second
-
-// Strongly typed address.
-type draAddr string
+const (
+	highestAvailableGRPCApi = "v1alpha3"
+	PluginClientTimeout     = 10 * time.Second
```

Comment:
Would it make sense to set this timeout to a half of the time between [Un]PrepareResources is called by the Kubelet? As @klueska mentioned in [this issue](https://github.com/kubernetes/kubernetes/issues/117146) it's around 90s. It's better to find out that value/constant and use it here.

Reply:
Yes. We should do that.

Reply:
Not sure to understand, you want the timeout to be set to 5 seconds ? or having two different timeout one for preparing and another for unpreparing ?

Reply:
Sorry for not being clear. Let me give a little bit more details on this. 
When [Un]PrepareResources call is failed, the Kubelet repeats it after some time (around 90 sec). I proposed to find out how often it happens and set PluginClientTimeout to a half of that value.

Reply:
Best would be to import the variable that defines this and just set the local timeout to this value / 2